pybossa/hateoas.py

```diff
@@ -38,7 +38,16 @@ def create_links(self, item):
         """Create Hateoas links."""
         cls = item.__class__.__name__.lower()
         links = []
-        if cls == 'taskrun':
+        if cls == 'result':
+            link = self. create_link(item.id, title='result')
+            if item.project_id is not None:
+                links.append(self.create_link(item.project_id, title='project',
+                                              rel='parent'))
+            if item.task_id is not None:
+                links.append(self.create_link(item.task_id, title='task',
+                                              rel='parent'))
```

Comment:
It should, but I didn't want to add them here because if a task has 100 task_runs, then, there'll be 100 links. It was too much I think, and it will affect the performance. What do you think?